docs/conf.py

```diff
@@ -295,3 +295,12 @@
 # ------------------------------------
 
 hoverxref_auto_ref = True
+hoverxref_project = "scrapy"
+hoverxref_version = release
+hoverxref_role_types = {
+    "class": "tooltip",
+    "confval": "tooltip",
+    "hoverxref": "tooltip",
+    "mod": "tooltip",
+    "ref": "tooltip",
+}
```

Comment:
We might want to add our custom roles as well, based on https://github.com/scrapy/scrapy/issues/4475#issuecomment-613350667

Reply:
Created https://github.com/scrapy/scrapy/issues/4495 for this.